### PR TITLE
feat: refactored logs stream to allow to be passed and initialized la…

### PR DESCRIPTION
**Pull request by exu**
…ter with id

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vLia — APPROVED** [review]
Looks good! 👍🏻

File: pkg/logs/client/stream.go [hunk 1]
```diff
@@ -15,75 +15,71 @@ import (
...
 }
 
 // Get returns channel with log stream chunks for given execution id connects through GRPC to log service
-func (c NatsLogStream) Get(ctx context.Context) (chan events.LogResponse, error) {
+func (c NatsLogStream) Get(ctx context.Context, id string) (chan events.LogResponse, error) {
 	ch := make(chan events.LogResponse)
 
-	name := fmt.Sprintf("lc%s%s", c.id, utils.RandAlphanum(6))
-	cons, err := c.js.CreateOrUpdateConsumer(ctx, c.streamName, jetstream.ConsumerConfig{
+	name := fmt.Sprintf("lc%s%s", id, utils.RandAlphanum(6))
```
Comment (vsukhin) [inline pkg/logs/client/stream.go:81]: is this const?

File: pkg/logs/client/stream.go [hunk 2]
```diff
@@ -123,12 +119,16 @@ func (c NatsLogStream) Get(ctx context.Context) (chan events.LogResponse, error)
 }
 
 // syncCall sends request to given subject and waits for response
-func (c NatsLogStream) syncCall(ctx context.Context, subject string) (resp StreamResponse, err error) {
-	b, _ := json.Marshal(events.Trigger{Id: c.id})
+func (c NatsLogStream) syncCall(ctx context.Context, subject, id string) (resp StreamResponse, err error) {
+	b, _ := json.Marshal(events.Trigger{Id: id})
```
Comment (vsukhin) [inline pkg/logs/client/stream.go:123]: i perefer to check all errors

**Review by vsukhin — APPROVED** [review]
minor comments

**Pull request merged**